firmware/rhizome.ino

```diff
@@ -80,5 +90,13 @@ void loop() {
  * Delegation function that allows us to call the managed method for publishing periodic updates.
  */
 void doPeriodicUpdates() {
-    sproutList.publishPeriodicUpdates();
+	//Do not attempt to publish updates if disconnected from the cloud
+    if(Particle.connected() == false){
+		if(EEPROM.read(1) == 0x01){
```

Comment:
Agreed... I planned on defining it in the Menu object, but extending EEPROM makes sense too..